### PR TITLE
Improve error information - follow up Part I

### PR DESCRIPTION
**Pull request by christianguenter2**
- Bugfix: remove errorenous regex
- Add (raw message) tooltip for message id and message class

#2811

### CODE REVIEW

File: src/ui/zcl_abapgit_gui_chunk_lib.clas.abap [hunk 1]
```diff
@@ -218,12 +209,21 @@ CLASS zcl_abapgit_gui_chunk_lib IMPLEMENTATION.
...
+      lv_title = get_t100_text(
+                    iv_msgid = ix_error->if_t100_message~t100key-msgid
+                    iv_msgno = ix_error->if_t100_message~t100key-msgno ).
+
       ro_html->add_a(
-          iv_txt = lv_msgid_and_msgno
-          iv_typ = zif_abapgit_html=>c_action_type-sapevent
-          iv_act = zif_abapgit_definitions=>c_action-goto_message
-          iv_id  = `a_goto_message` ).
+          iv_txt   = lv_msgid_and_msgno
```
Comment (sbcgua) [inline src/ui/zcl_abapgit_gui_chunk_lib.clas.abap]: What about changing this to just 'Message' or 'Message ({msgid}/{msgno})' ? The msg number does not say anything to a user. And those who are interested will click and see all the details. Imho just 'Message' is enough and it's number can be concatenated into title.

Comment (christianguenter2) [reply src/ui/zcl_abapgit_gui_chunk_lib.clas.abap]: image

Comment (sbcgua) [reply src/ui/zcl_abapgit_gui_chunk_lib.clas.abap]: Looks fine :)

**Comment by larshp** [comment]
cool, merging

**Pull request merged**